tests/trainer/logging/test_eval_loop_logging_1_0.py

```diff
@@ -198,10 +199,11 @@ def validation_epoch_end(self, outputs):
     assert pbar_metrics == expected_pbar_metrics
 
     callback_metrics = set(trainer.callback_metrics.keys())
+    callback_metrics.remove('debug_epoch')
```

Comment:
anyone knows what this is and what it is needed for?

Reply:
Internal debugger https://github.com/PyTorchLightning/pytorch-lightning/blob/cf9fe4905eac1bb66a8b72c7799cf77d6f6450f3/pytorch_lightning/trainer/connectors/logger_connector.py#L102-L109